contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -433,18 +448,20 @@ var EditMetadataEditor = BaseViews.BaseView.extend({
     });
   },
   load_license:function(){
+    iscopied = this.selected_items.length === 1 && !this.selected_items[0].isoriginal
     var license_modal = new LicenseModalView({
-      select_license : window.licenses.get({id: $("#license_select").val()})
+      select_license : window.licenses.get({id: (iscopied)? this.selected_items[0].model.get("license") : $("#license_select").val()})
     })
   },
   update_count:function(){
-    if(this.$("#input_description").is(":visible")){
+    if(this.selected_items.length === 1){
       stringHelper.update_word_count(this.$("#input_description"), this.$("#description_counter"), this.description_limit);
     }
   },
   add_tag: function(event){
     $("#tag_error").css("display", "none");
-    if((!event || (!event.keyCode || event.keyCode ==13)) && this.$el.find("#tag_box").length > 0 && this.$el.find("#tag_box").val().trim() != ""){
+    var code = (!event)? null : event.keyCode ? event.keyCode : event.which;
```

Comment:
Need to check event.keyCode or event.which depending on browser: https://trello.com/c/2kHe2hzs/212-tags-only-accept-one-letter